acceptance/setup/aio/pre-suite/020_AIO_Workarounds.rb

```diff
@@ -4,10 +4,10 @@
 
 step "(PUP-4004) Set permissions on puppetserver directories that currently live in the agent cache dir"
 %w[reports server_data yaml bucket].each do |dir|
-  on master, "install --directory /opt/puppetlabs/puppet/cache/#{dir}"
+  on master, "install --directory #{options[:HOSTS][:master][:puppetvardir]}/#{dir}"
```

Comment:
Ideally, I hope we can do:

```
vardir = master.puppet('master')['vardir']
```
This will effectively run `puppet master --configprint vardir` on the master node, so that we don't need to rely on `vardir` being set in the host yaml files (or beaker). This assumes that the master is setting the correct values in the shared `puppet.conf` and not in some other config file that puppet (ruby) can't see.